chroma-agent/chroma_agent/agent_client.py

```diff
@@ -279,7 +279,10 @@ def terminate(self, plugin_name):
         else:
             daemon_log.info("SessionTable.terminate %s/%s" % (plugin_name, session.id))
             session.teardown()
-            del self._sessions[plugin_name]
+            try:
+                del self._sessions[plugin_name]
+            except KeyError:
+                daemon_log.warning("SessionTable.terminate session object already gone")
```

Comment:
name is logged on line 280 which is not as good as doing it on the same line but should be able to work it out in the logs